tests/ops/functions/test_assert_valid.py

```diff
@@ -342,7 +342,11 @@ def create_op_instance(c, str_wires=False):
     if dim == 0:
         params = [1] * len(ndim_params)
     elif dim == 1:
-        params = [[1] * 2**n_wires] * len(ndim_params)
+
+        if c == qml.QubitStateVector:
+            params = [[1] + [0] * (2**n_wires - 1)] * len(ndim_params)
+        else:
+            params = [[1] * 2**n_wires] * len(ndim_params)
```

Comment:
Why this change is needed in a deprecation PR?

Reply:
The reason is a bit hidden. I had to remove `QubitStateVector` from `tests/ops/functions/conftest.py` because the deprecation warning makes it unable to instantiate.  By removing it from there, the operator will be tested in `def create_op_instance` (this test). This function define some parameters “valid for all templates” but it is not valid for `QubitStateVector`. So I have to manually give a definition of valid parameters

Reply:
Instead, we can add `QubitStateVector` to either `tests/ops/functions/conftest.py` `_ABSTRACT_OR_META_TYPES` or `_INSTANCES_TO_FAIL`.  Then it will just be skipped during this automatic testing process.